pkg/cmd/server/admission/init.go

```diff
@@ -38,3 +40,19 @@ func Validate(plugins []admission.Interface) error {
 	}
 	return nil
 }
+
+// GetPluginConfigFile translates from the master plugin config to a file name containing
+// a particular plugin's config (the file may be a temp file if config is embedded)
+func GetPluginConfigFile(pluginConfig map[string]api.AdmissionPluginConfig,
```

Comment:
This doesn't look like the correct package for this function.  Why not put it with the function you're delegating to?